### DISCUSSION TITLE
Jdownloader container in Synology no longer has Gui

### DISCUSSION THREAD

**Discussion by Harryzwo** [jlesage/docker-jdownloader-2#237] [General]
I get the following error
![Screenshot Jdownloader](https://github.com/user-attachments/assets/ee28de9b-04f5-4ad4-b0ad-e0b980606c94)
The container itself appears to be running. A restart is unsuccessful.

**Comment by jlesage** [comment]
Try the following method to see if it helps: https://github.com/jlesage/docker-jdownloader-2?tab=readme-ov-file#jdownloader-fails-to-start

**Comment by Harryzwo** [comment]
Super. That worked. Thanks! Should I delete the .fix_jd_install file again now?

**Comment by jlesage** [comment]
The file should be removed automatically.
